src/things/mod.rs

```diff
@@ -23,10 +23,22 @@ use tracing::instrument;
 use crate::cli::Config;
 use async_trait::async_trait;
 
+// Reddit has a new rate limit as of 7/1/2023:
+// OAuth for authentication: 100 queries per minute per OAuth client id - sleep atleast 0.6s after every call ( 650 ms)
+// not using OAuth for authentication: 10 queries per minute - must sleep atleast 6s between calls ( 6500 ms)
+const SLEEP_TIME:u64 = 650;
+const SLEEP_DUR:Duration = Duration::from_millis(SLEEP_TIME);
+pub async fn prevent_rate_limit() { 
+    println!("\n -- Sleeping for {SLEEP_TIME}ms to prevent rate limiting...\n");
```

Comment:
No need for `\n` in `println!` (the `ln` in `println` means it adds line breaks itself). Also could you update this to use `debug!` and remove the `-- `?

Reply:
done.